src/services/wallet/index.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+import { BigNumber } from 'bignumber.js';
+import factory from '../../services/ethereum/factory';
+import type { WalletType } from '../../types/Wallet';
+
+export default class WalletService {
+  static async getWallets() {
+    const { Panthalassa } = NativeModules;
+    const walletAddress = await Panthalassa.PanthalassaEthAddress();
+    const walletArray = [{
+      currency: 'ETH', ethAddress: walletAddress, balance: null, name: 'Ethereum',
+    }, {
+      currency: 'PAT', ethAddress: walletAddress, balance: null, name: 'XPAT',
+    }];
+    return walletArray;
+  }
+
+  static async syncWallet(wallet: WalletType) {
+    // const container = await containerPromise;
+    // return await container.eth.wallet.ethSync(wallet.ethAddress);
+    return wallet;
+  }
+
+  static async resolveBalance(wallets: Array<WalletType>, network: string) {
+    const walletsToCheck: Array<WalletType> = wallets;
+    const { Panthalassa } = NativeModules;
+    const walletAddress = await Panthalassa.PanthalassaEthPrivateKey();
+    const ethereum = await factory({ private_key: `0x${walletAddress}`, provider_type: network === 'dev' ? 'rinkeby' : 'homestead' });
+    const ethService = ethereum.service;
+
+    try {
+      const balance = await ethService.getBalance();
+      BigNumber.config({ DECIMAL_PLACES: 5 });
```

Comment:
I'm not sure why do you need that if you then round to 5 digits.

Reply:
This was taken from the `PrettyBalance` formatter, so it looked the way it was on the App. But it's open to change if you think so

Reply:
Ok, fixed